rastervision/backend/torch_utils/chip_classification/data.py

```diff
@@ -74,13 +103,28 @@ def build_databunch(data_dir, img_sz, batch_sz, class_names, augmentors):
     valid_ds = AlbumentationDataset(
         ImageFolder(valid_dir, classes=class_names))
 
+    if rare_classes != []:
+        targets = [target[1] for target in train_ds.orig_dataset.imgs]
```

Comment:
It'd be more concise (and also make fewer assumptions about the private variables of `train_ds`) if you just used `[target for _, target in train_ds]`.

Reply:
Resolved as you proposed